rclrs/src/client.rs

```diff
@@ -159,17 +159,17 @@ where
     /// signatures and which returns a `()` (a.k.a. nothing).
     /// ```
     /// # use rclrs::*;
-    /// # use rclrs::vendor::test_msgs::srv::{Empty, Empty_Request, Empty_Response};
+    /// # use crate::rclrs::vendor::test_msgs;
```

Comment:
I'm a bit confused about the leading `crate::` here since I thought doctests give a compilation error when you do `use crate::`. But I guess if the tests are passing then :shrug:

Reply:
I don't think we even need `crate::`. Locally, doctests seem to be passing with out it